plugins/Referrers/DataTable/Filter/UrlsForSocial.php

```diff
@@ -43,7 +43,6 @@ public function filter($table)
 
         // prettify the DataTable
         $table->filter('ColumnCallbackReplace', array('label', 'Piwik\Plugins\Referrers\removeUrlProtocol'));
-        $table->filter('Sort', array(Metrics::INDEX_NB_VISITS, 'desc', $naturalSort = false, $this->sortRecursive));
```

Comment:
No longer needed since we do now always sort in GenericFilter. So we do no longer sort the entire table twice